_common/qiskit/execute.py

```diff
@@ -254,6 +253,7 @@ def set_execution_target(backend_id='qasm_simulator',
                         provider_name='Quantinuum')
     """
     global backend
+    global sampler
```

Comment:
move the location of the global variable declaration of `sampler` (used to be in line `313`)